README.md

```diff
@@ -1 +1,198 @@
-# To-do-list-by-Kashif
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instrsuctions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+=======
+
+## Awesome books Project
+
+This project is about a basic website that allows users to add/remove books from a list. We will achieve that by using JavaScript objects and arrays. 
+We will also dynamically modify the DOM and add basic events.
+
+
+
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="/images/murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you fix this logo.

  ![idid](https://user-images.githubusercontent.com/36542986/206104859-62c264d4-7d71-4e86-a92f-50b465928970.jpg)